source/recording-interviews.md

```diff
@@ -1,10 +1,161 @@
+appropriate parameter. Audio quality is directly related to the file size that
+will be stored on the server. Keep in mind how much of your total storage space
+you want to use towards your audio files. Refer to the table below when choosing
+the appropriate parameter:
+
+| Quality    | Parameters         | Extension | Encoding | Bit rate  | Sample rate | File size    |
+| ---------- | ------------------ | --------- | -------- | --------- | ----------- | ------------ |
+| normal     | quality=normal     | .m4a      | AAC      | 64 kbps   | 32 kHz      | ~ 30 MB/hour |
+| low        | quality=low        | .m4a      | AAC      | 24 kbps   | 32 kHz      | ~ 11 MB/hour |
+| voice-only | quality=voice-only | .amr      | AMR      | 12.2 kbps | 8 kHz       | ~ 5 MB/hour  |
+
+You can leave the column blank to have the parameter set to `voice-only`, which
+will capture audio well in a quiet interview setting. If you are recording audio
+where there could be multiple people talking simultaneously, `low` would be more
+suitable. `normal` is the highest quality option and it will use the most
+storage space.
+
+## Collecting background audio with Collect Android app
+
+Review our support article,
+[Data Collection on KoBoCollect App](kobocollect_on_android_latest.md), to learn
+in detail about collecting data on the **Collect Android app**.
+
+<video controls>
+  <source
+    src="./_static/files/recording_interviews/collecting_data_with_background_audio_in_collect_app.mp4"
+    type="video/mp4"
+  />
+</video>
+
+While actively recording background audio with the **Collect Android app**, you
+should be able to see a microphone at the top of your form.
+
+![Background audio screen](/images/recording_interviews/background_audio_screen.jpg)
+
+## Viewing audio files that were recorded as background audio
+
+When you have `background-audio` configured for your project, you can view the
+recorded audio file under the **DATA>Table** as shown in the image below.
+
+![Data table view](/images/recording_interviews/data_table_view.png)
+
+## Downloading audio files
+
+You can download all background audio files as a ZIP file from
+**DATA>Downloads>Media Attachments (ZIP)** as shown in the video below.
+
+<video controls>
+  <source
+    src="./_static/files/recording_interviews/downloading_audio_files_that_were_recorded_as_background_audio.mp4"
+    type="video/mp4"
+  />
+</video>
+
+## Ethical considerations
+
+When collecting data, it is ethical to have informed consent from survey
+respondents prior to data collection, in this case by recording background
+audio.
+
+<p class="note">
+  We encourage all users to consider ethical implications of their data
+  collection and to comply with applicable data protection legislation within
+  the jurisdiction of their work.
+</p>
+
+## Troubleshooting
+
+-   This feature is supported with the **Collect Android app** `v1.30.0` and
+    above.
+-   This feature is currently not supported in the **Enketo web forms**
```

Comment:
`.` missing in the end of **Enketo web forms** so it should be **Enketo web forms**.